prerendering.bs

```diff
@@ -478,172 +459,146 @@ Every {{Document}} has an <dfn for="Document">activation start time</dfn>, which
 
-  1. If <var ignore>browsingContext</var> is a [=prerendering browsing context=] and <var ignore>browsingContext</var>'s [=prerendering browsing context/remove from initiator=] is not null, then call <var ignore>browsingContext</var>'s [=prerendering browsing context/remove from initiator=].
+  1. If |traversable| is a [=prerendering traversable=] and |traversable|'s [=prerendering traversable/remove from referrer=] is not null, then call |traversable|'s [=prerendering traversable/remove from referrer=].
 </div>
 
-<h3 id="creating-bcs-patch">Modifications to creating browsing contexts</h3>
+<h3 id="creating-navigables-patch">Modifications to creating navigables</h3>
 
-<div algorithm="create a new nested browsing context patch">
-  To ensure that any [=nested browsing contexts=] inherit their parent's [=browsing context/loading mode=], modify <a spec=HTML>create a new nested browsing context</a> by appending the following step:
+<div algorithm="create a new nested navigable patch">
+  To ensure that any [=child navigables=] inherit their [=navigable/parent=]'s [=navigable/loading mode=], modify [=create a new nested navigable=] by appending the following step:
 
-  1. Set <var ignore>browsingContext</var>'s [=browsing context/loading mode=] to <var ignore>element</var>'s [=Node/node document=]'s [=Document/browsing context=]'s [=browsing context/loading mode=].
+  1. Set <var ignore>navigable</var>'s [=navigable/loading mode=] to <var ignore>element</var>'s [=node navigable=]'s [=navigable/loading mode=].
 </div>
 
 <h2 id="navigation">Navigation and session history</h2>
 
 <h3 id="navigate-activation">Allowing activation in place of navigation</h3>
 
-<div algorithm="can activate a prerender">
-  We <dfn>can activate a prerender</dfn> given a [=browsing context=] |browsingContext|, a [=history handling behavior=] |historyHandling|, a string |navigationType|, and a [=request=] |request|, if the following steps return true:
+<div>
+  To <dfn>get the matching prerendering navigable</dfn> given a [=navigable=] |navigable|, a [=URL=] |url|, a string |cspNavigationType|, and a |documentResource|, if the following steps return true:
 
-  1. Return true if all of the following are true:
+  1. If any of the following conditions hold:
 
-      * |browsingContext| is a [=top-level browsing context=]
-      * |browsingContext| is not a [=prerendering browsing context=]
-      * |browsingContext| is not for a [Fenced Frame](https://github.com/shivanigithub/fenced-frame)
-         <p class="issue">The concept of a fenced frame browsing context is not yet defined but we need to link to it once it exists.</p>
-      * |historyHandling| is "`default`" or "`replace`"
-      * |navigationType| is "`other`"
-      * |request|'s [=request/method=] is \``GET`\`
```

Comment:
Is this obsoleted by the |documentResource| check? If so, why? I'm guessing that GET requests don't have them, POST requests always do, and no other type is possible in navigation? But it's not obvious.

Reply:
Yes. Every POST request will have a POST resource as |documentResource|, in particular. So that subsumes the old check.

The case where |documentResource| is a string, occurs when navigating srcdoc iframes. We also want to bail out there, although we should never have a non-HTTP(S) URL in the map anyway.

Reply:
It wasn't totally obvious to me that a "document resource" refers to a POST response body (and associated metadata), though now that I know that it's clear. Could go either way.